qa/sc_versions_and_mc_certs.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import codecs
+import pprint
+import time
+from decimal import Decimal
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.mc_test.mc_test import CertTestUtils, generate_random_field_element_hex
+from test_framework.util import assert_false, start_nodes, \
+    websocket_port_by_mc_node_index
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, generate_next_blocks, generate_next_block, \
+    assert_true, create_alien_sidechain, create_certificate_for_alien_sc
+from SidechainTestFramework.sc_forging_util import *
+
+"""
+Configuration:
+    Start 1 MC node and 1 SC node (with default websocket configuration).
+    SC node connected to the first MC node.
+
+Test:
+    TODO    
```

Comment:
need to add description.
Test itself seems fine.

Reply:
Done